<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>980</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>To-Do List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(0, 0, 0)</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="due">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>200</y>
      <width>650</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255,255,255);
border-bottom: 2px solid white;
border-left: 0px;
border-top: 0px;
border-right: 0px;
}

QLineEdit:focus {
color:rgb(255,255,255);
border-bottom: 2px solid rgb(48, 159, 255);
border-left: 0px;
border-top: 0px;
border-right: 0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Due Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="task_in">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>140</y>
      <width>650</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
color:rgb(255,255,255);
border-bottom: 2px solid white;
border-left: 0px;
border-top: 0px;
border-right: 0px;
}

QLineEdit:focus {
color:rgb(255,255,255);
border-bottom: 2px solid rgb(48, 159, 255);
border-left: 0px;
border-top: 0px;
border-right: 0px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>340</y>
      <width>650</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(48, 159, 255);
color:white;
border-radius: 20px;
}
QPushButton:hover{
background-color:rgb(37, 131, 255);
color:white;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>280</y>
      <width>650</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(48, 159, 255);
color:white;
border-radius: 20px;
}
QPushButton:hover{
background-color:rgb(37, 131, 255);
color:white;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="warn">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>10</y>
      <width>661</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255)</string>
    </property>
    <property name="text">
     <string>Warnings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1231</width>
      <height>961</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Pending Tasks:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="task_out">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(215, 215, 215);
border-radius: 10px;

</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>70</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Add Task:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
